### PR TITLE
Documentation for the ZIS messages

### PR DESCRIPTION
**Pull request by ifakhrutdinov**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
This pull-request adds documentation for the ZIS, ZIS AUX, and ZIS dynamic linkage base plugin messages.

:heart:Thank you!

### CODE REVIEW

**Comment by ifakhrutdinov** [comment]
FYI, doc squad members, the link [guidelines for contributing](https://github.com/zowe/docs-site/docs/contribute/contributing.md) in the PR template seems to be broken.

File: docs/troubleshoot/app-framework/zis-error-codes.md [hunk 1]
```diff
@@ -0,0 +1,1985 @@
...
+
+**Reason:**
+
+The cross-memory auxiliary server detected that it was running in the wrong key.
+
+**Action:**
+
+The cross-memory auxiliary server terminates. Ensure that you have added the
+correct PPT-entry (
+see [the documentation](https://docs.zowe.org/stable/user-guide/configure-xmem-server/))
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/zis-error-codes.md:393]: These types of URLs could break if someone changes the doc tech.
Instead, try relative:
(../../user-guide/configure-xmem-server.md)

you will see examples of this elsewhere, https://github.com/ifakhrutdinov/docs-site/blob/feature/zis-message-ids/docs/user-guide/systemrequirements-zos.md?plain=1#L99

File: docs/troubleshoot/app-framework/zis-error-codes.md [hunk 2]
```diff
@@ -0,0 +1,1985 @@
...
+
+**Reason:**
+
+The cross-memory server detected that it was running in the wrong key.
+
+**Action:**
+
+The cross-memory server terminates. Ensure that you have added the correct
+PPT-entry (
+see [the documentation](https://docs.zowe.org/stable/user-guide/configure-xmem-server/))
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/zis-error-codes.md:1023]: These types of URLs could break if someone changes the doc tech.
Instead, try relative:
(../../user-guide/configure-xmem-server.md)

you will see examples of this elsewhere, https://github.com/ifakhrutdinov/docs-site/blob/feature/zis-message-ids/docs/user-guide/systemrequirements-zos.md?plain=1#L99

Comment (ifakhrutdinov) [reply docs/troubleshoot/app-framework/zis-error-codes.md]: Fixed.

File: docs/troubleshoot/app-framework/zis-error-codes.md (hunk 1 above)
Comment (ifakhrutdinov) [reply docs/troubleshoot/app-framework/zis-error-codes.md]: Fixed.

**Comment by ifakhrutdinov** [comment]
@1000TurquoisePogs whom else do we need to involve to move this forward?

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**